src/Shell/Helper/TableHelper.php

```diff
@@ -78,6 +78,10 @@ protected function _rowSeparator($widths)
      */
     protected function _render($row, $widths, $options = [])
     {
+        if(!$row || !is_array($row)) {
```

Comment:
`empty($rows)` or `count($rows)` would be shorter. Additionally, if rows is not an array it has the nice side effect that you will get an error :)